packages/starlight/components/SidebarSublist.astro

```diff
@@ -17,7 +17,8 @@ interface Props {
 					<a
 						href={entry.href}
 						aria-current={entry.isCurrent && 'page'}
-						class:list={{ large: !Astro.props.nested }}
+						class:list={[{ large: !Astro.props.nested }, entry.attrs?.['class']]}
```

Comment:
For example here we’d be able to do this:
```suggestion
						class:list={[{ large: !Astro.props.nested }, entry.attrs.class]}
```

Reply:
Oh yeah, I'm way to used to `noPropertyAccessFromIndexSignature` ^^